### DISCUSSION TITLE
Complete Edition Variant - Work Phases

### DISCUSSION THREAD

**Discussion by SolitarYo93** [Taker597/WitcherOldWorldComplete#18] [General]
We should, sooner or later, give us a raw schedule based on Taker597 phases proposal.

> Phase 1 - Playtesting of base game, Data Collecting, and Open Discourse of Complete Edition Varient ideas
> Phase 2 - Collectively decide and outline what Varient ideas works best within the board game space that's streamline,balance, and won't add unnecessary length to the playtime.
> Phase 3 - Drafting Complete Edition
> Phase 4 - Playtest Complete Edition
> Phase 5 - Implement Fixes and further balance
> Phase 6 - Finalize, Publish pdf files, and Complete project

Here instead Nanadav2 proposal.

> Here is another perspective on how you could start approaching this
> Phase 1 - Ideation. What do we need to change? Is there a wide consensus on the elements that should and should not be used? Elementa of the game that need to change? It may be worth not just listing these, but maybe even creating some way for folks to vote on what changes (from their experience) they think should be reworked from the base game.
> 
> Later phases could be to outsource some of these decisions to other people in the community and either reuse variants that others have already suggested to suggest new ones and try to get feedback/ideas from the community including playtesting what works and what doesn't.
> 
> Combining these ideas together will be a difficult, interesting and important last step.
> 
> Thoughts?
> But as I said, for that you should create some external document to show who is working on what, and to have a way for folks to express opinions/feedback/vote on specific sub-elements we're working (or should) work on.
> 
> I can also help with setting this up if you need ideas on the technical side of things

I would start deciding all togheter a well defined path and then start detailing each phase, giving an ideal timeline, define who work on each step and milestone.
What's important here is to don't get lost.

**Comment by Taker597** [comment]
I would like to wait and see if we can recruit atleast two more members before designating roles and timelines. As it stands... I like to complete the project within 4 months.

**Reply by SolitarYo93** [reply]
Indeed we have to wait and see
4 months it's a good deadline i think, anyway we need all the help we can get.
